packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -368,7 +371,10 @@ export class BlobManager extends TypedEventEmitter<IBlobManagerEvents> {
         return PerformanceEvent.timedExecAsync(
             this.mc.logger,
             { eventName: "createBlob" },
-            async () => this.getStorage().createBlob(blob),
+            async () => {
+                const res: ICreateBlobResponse & Partial<Record<"minTTLInSeconds", number>> = await this.getStorage().createBlob(blob);
```

Comment:
since we use this composed type twice, it might be nice to create a type to use in this file rather than repeating.
`const type ICreateBlobResponseWithTTL = ICreateBlobResponse & Partial<Record<"minTTLInSeconds", number>>`